rio_alpha/alpha.py

```diff
@@ -3,6 +3,18 @@
 import riomucho
 from rio_alpha.alpha_mask import mask_exact
 
+# Convert window to an instance of rasterio.windows.Window
+# if possible, to avoid Rasterio deprecation warnings.
+#
+# The window_guard function is used like:
+#
+# window = window_guard(window)
```

Comment:
👍  would it be worth adding this function to rasterio (similar to `guard_transform`)?

Reply:
Well, we can't put it in older versions of rasterio, so it kinda needs to be in plugins or user code. An alternative would be to require `rasterio>=1.0a10` here, but depending on pre-releases is a bit complicated for some users.